darshan-util/pydarshan/tests/test_log_utils.py

```diff
@@ -0,0 +1,55 @@
+import os
+
+import darshan
+from darshan.log_utils import get_log_path
+
+import pytest
+
+
+@pytest.mark.parametrize("log_filename", [
+    "noposixopens.darshan",
+    "sample-badost.darshan",
+    "sample-dxt-simple.darshan",
+    "noposix.darshan",
+    "sample-goodost.darshan",
+    "sample.darshan",
+    "pq_app_read_id71344_7-31-5658-2037904274838284930_55623.darshan",
+    "pq_app_readAB_writeC_id71326_7-31-5658-2037904274838284930_55623.darshan",
+    "pq_app_write_id71296_7-31-5657-2037904274838284930_55623.darshan",
+    "pq_app_write_id71310_7-31-5657-2037904274838284930_55623.darshan",
+    "pq_app_read_id71317_7-31-5657-2037904274838284930_55623.darshan",
+    "pq_app_write_id71303_7-31-5657-2037904274838284930_55623.darshan",
```

Comment:
Do we want to include these files here? We *could* manipulate the glob search to remove them since they all start with the same name: `p.glob('**/[!pq_app_]*.darshan')`, but not a big deal either way.

Reply:
I suspect if we really don't want to glob them in (i.e., they have no possible use, and they may not?), we should just delete them. I'll merge this as is then and if the team wants to dump those files after that's fine by me.